src/views/newConnection.tsx

```diff
@@ -26,7 +26,9 @@ export default class newConnectionTemplates {
         <div class="connections header">
           <span>Invite New Connection</span>
         </div>
-        <this.companyFormInput targetBox={targetBox} formStage={formStage} />
+        <div class="connections list ">
```

Comment:
this looks a bit silly. I assume this wasn't mean to be two classes but instead one called `connection-list`?

Reply:
this was sth Paul was using I left it be and created a new class `card-body` which I'll be using